lua/lazyvim/util/ui.lua

```diff
@@ -141,9 +141,17 @@ function M.statuscolumn()
   local is_relnum = vim.wo[win].relativenumber
   if (is_num or is_relnum) and vim.v.virtnum == 0 then
     if vim.v.relnum == 0 then
-      components[2] = is_num and "%l" or "%r" -- the current line
+      if vim.fn.has("nvim-0.11") == 1 then
+        components[2] = "%l" -- the current line
+      else
+        components[2] = is_num and "%l" or "%r" -- the current line
+      end
     else
-      components[2] = is_relnum and "%r" or "%l" -- other lines
+      if vim.fn.has("nvim-0.11") == 1 then
+        components[2] = "%l" -- other lines
+      else
+        components[2] = is_relnum and "%r" or "%l" -- other lines
+      end
     end
```

Comment:
I feel like simplifying this to the following more clearly separates the behaviour of the new change
```lua
    if vim.fn.has("nvim-0.11") == 1 then
      components[2] = "%l" -- 0.11 handles both the current and other lines with %l
    else
      if vim.v.relnum == 0 then
        components[2] = is_num and "%l" or "%r" -- the current line
      else
        components[2] = is_relnum and "%r" or "%l" -- other lines
      end
    end
```

Reply:
@bryanniwa Thank you very much for improving and simplifying the logic even more. 

Just pushed the new changes you suggested :smile: